core/metacling/src/TCling.cxx

```diff
@@ -1720,6 +1690,45 @@ namespace {
 
 }
 
+////////////////////////////////////////////////////////////////////////////////
+///\returns true if the module map was loaded, false on error or if the map was
+///         already loaded.
+bool TCling::RegisterPrebuiltModulePath(const std::string &FullPath,
```

Comment:
Why does this need to become a member now? I prefer to have *less* functions in `TInterpreter`...

Reply:
This is needed in Core: https://github.com/root-project/root/pull/3850/files#diff-849f40df79a2c791b5e8a7edd3845b81R3561